sympy/testing/tests/test_code_quality.py

```diff
@@ -147,6 +196,11 @@ def test(fname):
             _test_this_file_encoding(fname, test_file)
 
     def test_this_file(fname, test_file):
+        idx = line_with_bare_expr(test_file.read()) if 'bench' not in fname else None  # XXX what should be done about bench tests
```

Comment:
The benchmarks in the sympy repo are broken and should be removed so I think it's fine to skip them here.